dev/setup-cluster-config

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [[ -n "$1" && "$1" = "teardown" ]]; then
+    ./e2e/testenv/infra/infra teardown
+    exit 0
+fi
+
+if [ ! -f "$DEFAULT_CONFIG" ]; then
+    echo >&2 "Run this from the root of the repo"
+    exit 1
+fi
+
+if [ -n "$FLEET_TEST_CONFIG" ]; then
+    if [ ! -f "$FLEET_TEST_CONFIG" ]; then
+        echo >&2 "File not found: \$FLEET_TEST_CONFIG: $FLEET_TEST_CONFIG"
+        exit 1
+    fi
+    # shellcheck source=/dev/null
+    source "$FLEET_TEST_CONFIG"
+elif [ -f "$DEFAULT_CONFIG" ]; then
+    # shellcheck source=/dev/null
+    source "$DEFAULT_CONFIG"
+else
+    echo >&2 "File not found: \$DEFAULT_CONFIG: $DEFAULT_CONFIG"
+    exit 1
+fi
```

Comment:
The `else` block is unreachable, since we've already checked existence of `$DEFAULT_CONFIG` in L10-12, exiting when it is not found as a regular file.
```suggestion
else
    # shellcheck source=/dev/null
    source "$DEFAULT_CONFIG"
fi
```